browser/farbling/brave_screen_farbling_browsertest.cc

```diff
@@ -0,0 +1,203 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "base/path_service.h"
+#include "brave/browser/brave_content_browser_client.h"
+#include "brave/components/brave_shields/browser/brave_shields_util.h"
+#include "brave/components/constants/brave_paths.h"
+#include "chrome/browser/content_settings/host_content_settings_map_factory.h"
+#include "chrome/browser/extensions/extension_browsertest.h"
+#include "chrome/browser/ui/browser_window.h"
+#include "chrome/common/chrome_content_client.h"
+#include "chrome/test/base/ui_test_utils.h"
+#include "content/public/test/browser_test.h"
+#include "net/dns/mock_host_resolver.h"
+#include "net/test/embedded_test_server/embedded_test_server.h"
+#include "net/test/embedded_test_server/http_request.h"
+
+using brave_shields::ControlType;
+
+class BraveScreenFarblingBrowserTest : public InProcessBrowserTest {
```

Comment:
need to test with feature enabled and disabled

Reply:
Fixed